### PR TITLE
Remove unnecessary test helpers

### PR DESCRIPTION
**Pull request by naripok**
# Summary

This issue removes unnecessary helpers and abstractions from the edgekit's test suite.
Also, it makes some minor improvements in test code quality and clarity.

Part of https://github.com/AirGrid/rmap/issues/430

### CODE REVIEW

**Comment by ydennisy** [comment]
@naripok I think this looks good? Why draft?

**Comment by naripok** [comment]
> @naripok I think this looks good? Why draft?

I haven't finished looking through all the tests yet.

File: test/e2e/run.test.ts [hunk 1]
```diff
@@ -22,14 +24,16 @@ describe('edgekit basic run behaviour', () => {
   // look at jest-playwright.config.js
   const testUrl = 'http://localhost:9000';
 
+  const matchingVector = [1, 1, 1];
```
Comment (ydennisy) [inline test/e2e/run.test.ts:27]: @naripok do you think we could use this to create a rule: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/naming-convention.md#enforce-that-all-const-variables-are-in-upper_case

Which allows UPPER_CASE only for hardcoded values?

Comment (naripok) [reply test/e2e/run.test.ts:27]: Dude, the rule as it is there enforces UPPER_CASE for all the const variables, not only for the hardcoded ones (I've tested it).
I'll get some research done, however, to see if I can find anything suiting our needs.

File: test/helpers/engineConditions.ts [hunk 2]
```diff
@@ -21,22 +27,29 @@ export const makeEngineCondition = <T extends AudienceQueryDefinition>(
         name: 'count',
       },
       matcher: {
-        name: condition,
-        args: occurences,
+        name: condition || 'ge',
```
Comment (ydennisy) [inline test/helpers/engineConditions.ts:30]: the default we have is 'gt', I believe

File: test/helpers/engineConditions.ts [hunk 3]
```diff
@@ -21,22 +27,29 @@ export const makeEngineCondition = <T extends AudienceQueryDefinition>(
...
+  version: number;
+  ts?: number;
+  queryProperty?: string;
+};
+
+export const makePageView = ({
+  value,
+  version,
   ts,
+  queryProperty,
```
Comment (ydennisy) [inline test/helpers/engineConditions.ts:48]: this is the feature name.

File: test/helpers/engineConditions.ts [hunk 4]
```diff
@@ -21,22 +27,29 @@ export const makeEngineCondition = <T extends AudienceQueryDefinition>(
...
+}: MakePageViewInput): PageView => ({
+  ts: ts || 100,
   features: {
-    [queryProperty]: {
+    [queryProperty || 'docVector']: {
```
Comment (ydennisy) [inline test/helpers/engineConditions.ts:52]: Just force people to specify this, no need for the default.

File: test/unit/audience/cosineSimilarityQueryAudience.test.ts [hunk 5]
```diff
@@ -5,15 +5,49 @@ import {
...
-  cosineSimAudience,
-  multiCosineSimAudience,
+  makeAudienceDefinition,
+  makeCosineSimilarityQuery,
 } from '../../helpers/audienceDefinitions';
 
 describe('cosine similarity audiences matching behaviour', () => {
+  const vectorOne = [1, 1, 1];
+  const vectorTwo = [1, 0, 1];
+  const notMatchingVector = [0, 1, 0];
```
Comment (ydennisy) [inline test/unit/audience/cosineSimilarityQueryAudience.test.ts:15]: HARD_CODED :)

**Review by ydennisy — CHANGES_REQUESTED** [review]
Few small comments, the hardcoded lint rule maybe is worth adding to AG backlog and just making the changes here so we can land this with the other PR.

@naripok

**Pull request merged**